tools/ci_build/github/linux/build_cuda_c_api_package.sh

```diff
@@ -2,4 +2,4 @@
 set -e -x
 docker run -e SYSTEM_COLLECTIONURI --rm --volume \
 $BUILD_SOURCESDIRECTORY:/onnxruntime_src --volume $BUILD_BINARIESDIRECTORY:/build -e NIGHTLY_BUILD onnxruntimecuda${CUDA_VERSION_MAJOR}build \
-/bin/bash -c "/usr/bin/python3 /onnxruntime_src/tools/ci_build/build.py --enable_lto --build_java --build_nodejs --build_dir /build --config Release --skip_submodule_sync  --parallel --use_binskim_compliant_compile_flags --build_shared_lib --use_cuda --cuda_version=$CUDA_VERSION --cuda_home=/usr/local/cuda-$CUDA_VERSION --cudnn_home=/usr/local/cuda-$CUDA_VERSION --skip_tests --use_vcpkg --use_vcpkg_ms_internal_asset_cache --cmake_extra_defines 'CMAKE_CUDA_ARCHITECTURES=60-real;70-real;75-real;80-real;90a-real;90a-virtual' && cd /build/Release && make install DESTDIR=/build/installed"
+/bin/bash -c "/usr/bin/python3 /onnxruntime_src/tools/ci_build/build.py --enable_lto --build_java --build_nodejs --build_dir /build --config Release --skip_submodule_sync  --parallel --use_binskim_compliant_compile_flags --build_shared_lib --use_cuda --cuda_version=$CUDA_VERSION --cuda_home=/usr/local/cuda-$CUDA_VERSION --cudnn_home=/usr/local/cuda-$CUDA_VERSION --skip_tests --use_vcpkg --use_vcpkg_ms_internal_asset_cache --cmake_extra_defines 'CMAKE_CUDA_ARCHITECTURES=80-real;90a-real;90a-virtual' && cd /build/Release && make install DESTDIR=/build/installed"
```

Comment:
Do we have size limit on C api package? If not, I suggest to keep 70-real;75-real.

Reply:
Keeping either 70-real or 75-real will make the nuget package size over 250MB.
Do you know if there are customer still using 75-real (Turing) ?
Another option is we keep 75-real as there might be customer using it, but replace 90a-virtual by 120a-real to specifically for Blackwell and also reduce package size?

Reply:
> Keeping either 70-real or 75-real will make the nuget package size over 250MB. Do you know if there are customer still using 75-real (Turing) ? Another option is we keep 75-real as there might be customer using it, but replace 90a-virtual by 120a-real to specifically for Blackwell and also reduce package size?

We do not know 120a-real is smaller than 90a-virtual. That need upgrade cuda to 12.8+ in CI pipeline.

Reply:
With current package build configuration, Windows Python Wheel, Windows Nuget and Linux Python all have the support for SM75. Only Linux C API or Nuget is lacking it.

I can add back the support for Linux C API package.